File: src/main/java/NekoEssentials/commands/heal.java
package NekoEssentials.commands;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

import static NekoEssentials.Main.c;
import static NekoEssentials.Main.config;

public class heal implements CommandExecutor {

    @Override
    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
        if (label.equalsIgnoreCase("heal")) {
            if (sender instanceof Player) {
                Player player = (Player) sender;
                if (player.hasPermission("NekoEssentials.heal")) {
                    player.setHealth(player.getMaxHealth());
                    player.setFoodLevel(20);

                    player.sendMessage(c(config.getConfig().getString("healed")));
                    return true;
                } else {
                    player.sendMessage(c(config.getConfig().getString("no-permission")));
                }
            }
        }
        return false;
    }
}

File: src/main/java/NekoUtility/commands/heal.kt
package NekoUtility.commands

import NekoUtility.Main
import NekoUtility.Main.Companion.c
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class heal : CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (sender is Player) {
            val player = sender
            if (player.hasPermission("nekoutility.heal")) {
                player.health = player.maxHealth
                player.foodLevel = 20
                player.sendMessage(c(Main.config!!.getConfig().getString("healed")))
                return true
            } else {
                player.sendMessage(c(Main.config!!.getConfig().getString("no-permission")))
            }
        }
        return false
    }
}